content/en/docs/started/installing-kubeflow.md

```diff
@@ -153,6 +156,20 @@ The following table lists distributions which are <em>maintained</em> by their r
           <a href="https://nutanix.github.io/kubeflow-manifests">Website</a>
         </td>
       </tr>
+      <tr>
+        <td>
+          QBO
```

Comment:
Do you have maintainer name for this distribution of QBO is the company name ?

Reply:
QBO is the Company name or maintainer.